doc/migrations/3.1.1_3.1.1++.md

```diff
@@ -10,3 +10,26 @@
 ## **`ejabberd.cfg` renamed to `mongooseim.cfg`**
 
 Rename the existing config file of MongooseIM from `ejabberd.cfg` to `mongooseim.cfg`.
+
+## Pools configuration
+
+Configuring pools to external services has changed, please see [Outgoing Connection doc](../advanced-configuration/outgoing-connections.md) for more details.
+
+NOTE: Keep in mind that outgoing_pools is a list of pools, it may turn out that you will have more than one entry in the list when more than a single outgoing pool is needed.
+
+### ElasticSearch configuration migration
+
+Change the existing entry in the configuration file:
+
+```erlang
+{elasticsearch_server, [{host, "elastic.host.com"}, {port, 9042}]}.
+```
+
+to:
+
+```erlang
+{outgoing_pools, [
+ {elastic, global, elasticsearch, [], [{host, "elastic.host.com"}, {port, 9042}]}
```

Comment:
Why not `default` tag, to maintain convention?

Reply:
Mostly because internally in the 'tirerl' lib only the tag is passed as the pool name. To fix that I'd have to create the pool name from 'Type', 'Host' and 'Tag' every time there is a call to Elastic. I don't really like that, but on the other hand we already do it in some of the other pools.
This applies to other comments as well.